resources/views/Checkout/OrderDetails.twig

```diff
@@ -64,19 +64,38 @@
                     <div>
                         <div class="row">
                             <strong class="col-xs-6 col-sm-5">{{ trans("Ceres::Template.orderPaymentMethod") }}</strong>
-                            <span class="col-xs-6 col-sm-7">{{ orderData.paymentMethodName }}</span>
+                            <span class="col-xs-6 col-sm-7" id="payment_name_{{ orderData.order.id }}">{{ orderData.paymentMethodName }}</span>
                         </div>
                         <div class="row">
                             <strong class="col-xs-6 col-sm-5">{{ trans("Ceres::Template.orderPaymentStatus") }}</strong>
-
+                            {% set paymentStatus = '' %}
```

Comment:
Ich bin Hochsprachen gewöhnt in denen Variablen, die man in Schleifen deklariert, nicht ausserhalb dieser nutzen kann. Twig scheint das nicht zu haben. Kann also raus. Aber vorher bitte testen :) @chkolb